dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -54,25 +55,20 @@ def test_profiled_precision(self):
         df_3 = pd.Series([4.114, 3.161, 2.512, 2.131]).apply(str)
         df_mix = pd.Series([4.1, 3., 2.52, 2.13143]).apply(str)
 
-        float_profiler = FloatColumn(df_1.name)
-        float_profiler.update(df_1)
-        self.assertEqual(1, float_profiler.precision)
-
-        float_profiler.update(df_2)
-        self.assertEqual(2, float_profiler.precision)
-
+        float_profiler = FloatColumn("Name")
         float_profiler.update(df_3)
         self.assertEqual(3, float_profiler.precision)
 
         float_profiler.update(df_2)
-        self.assertEqual(3, float_profiler.precision)
+        self.assertEqual(2, float_profiler.precision)
 
-        float_profiler.update(df_mix)
```

Comment:
so we don't test df_mix ?

Reply:
oh, I see below. Should the precision be 5 as we have this number 2.13143. If the precision is 1, does that mean the number will be round up to 2.1?

Reply:
We are taking the lowest common denomination for precision. So the number is 1 since 3. exists in the dataset